Adicionar beautiful jekyll

jvfe opened the pull request:
Alterações feitas:
* Adiciona o template do beautiful jekyll para o github pages da organização.
* Adiciona uma página para as reuniões anteriores

jvfe committed:
Adiciona beautiful jekyll template

jvfe committed:
Limpa repositório e modifica index.md

jvfe committed:
Adiciona página de reuniões passadas

jvfe committed:
Identificar _config

Tira placeholders anônimos, identificando ele agora com o grupo do
código bonito.

jvfe committed:
Remove markdown desnecessário na raiz

jvfe committed:
Adiciona instruções para rodar localmente

jvfe committed:
Deleta changelog desnecessário

vinisalazar committed:
Tweak some config values for language, date formatting, and site title

vinisalazar committed:
Translate post date text

vinisalazar committed:
Edit landing page

vinisalazar committed:
Add posts in pt-br

vinisalazar committed:
Update .gitignore

vinisalazar committed:
Move pages to _pages folder.
    - Added permalinks to pages' frontmatter
    - Added 'reunioes' to navbar

vinisalazar committed:
Add avatar image

vinisalazar commented:
Eu fiz algumas mudanças em jvfe/codigobonito.github.io#1 que poderiam ser mergeadas antes desse PR.

Mandei as mudanças pelo repo original do @codigobonito mas futuramente vou enviar pelo meu fork.

jvfe committed:
Make a few improvements to the template #1


- Organize pages into a directory
- Set language attributes
- Modify navbar
- Modify site titles and subtitles

jvfe commented:
> Eu fiz algumas mudanças em [jvfe#1](https://github.com/jvfe/codigobonito.github.io/pull/1) que poderiam ser mergeadas antes desse PR.
> 
> Mandei as mudanças pelo repo original do @codigobonito mas futuramente vou enviar pelo meu fork.

Merge feito, valeu!

jvfe commented on docs/_config.yml:
```diff
@@ -52,28 +56,28 @@ social-network-links:
 #  twitch: yourname
 #  yelp: yourname
- telegram: codigobonito
+#  telegram: yourname
```
Eu já fiz o merge de jvfe#1 antes de perceber isso, mas eu tinha colocado o link pro grupo do telegram, visto que já é um grupo aberto, acho que não teria problema. O que vocês acham?

vinisalazar replied:
Por mim OK

vinisalazar replied:
@lubianat
